doc/api/rand.rst

```diff
@@ -1,79 +1,43 @@
 .. _openssl-rand:
 
-:py:mod:`rand` --- An interface to the OpenSSL pseudo random number generator
-=============================================================================
+:mod:`rand` --- An interface to the OpenSSL pseudo random number generator
+==========================================================================
 
-.. py:module:: OpenSSL.rand
-   :synopsis: An interface to the OpenSSL pseudo random number generator
-
-
-This module handles the OpenSSL pseudo random number generator (PRNG) and
-declares the following:
-
-.. py:function:: add(string, entropy)
+.. warning::
+   Functions from this methods shouldn't be used.
+   `Use urandom <http://sockpuppet.org/blog/2014/02/25/safely-generate-random-numbers/>`_ instead.
```

Comment:
We're not deprecating the whole module, that's just good-natured advice. :)

I wanna write a backwards compat doc before release (2 year after deprecation it goes to hell).